src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -949,53 +949,38 @@ class FrontendGenImpl {
   }
 
   std::string GetImportVersionOfNode(const onnx::NodeProto &node) {
-    auto schema = GetOpSchema(node);
-    // Assume the top version
-    if (schema == nullptr) {
-      return std::string("");
-    }
     auto current_opset = opset_map_.find(node.domain())->second;
 
     LLVM_DEBUG(llvm::dbgs() << DEBUG_TYPE << ": Importing ONNX"
                             << node.op_type() << " (" << node.name() << ")"
                             << ", Opset: " << current_opset << "\n");
 
+    auto opset_list_it = op_dialect_version_map_.find(node.op_type());
+
     // Custom ops may not be present in op_dialect_version_map_. If no version
     // info is found, treat as unversioned (no renaming).
-    auto opset_list_it = op_dialect_version_map_.find(node.op_type());
-    if (opset_list_it != op_dialect_version_map_.end()) {
-      auto opset_list = opset_list_it->second;
-      // A new opset is added to onnx-mlir when it becomes imcompactible.
-      // But the lowest opset in op_dialect_version_map_ is an exception.
-      // It is the current opset when onnx-mlir project is started.
-      // All opset lower than the last opset should use the last opset(version)
-
-      if (node.domain().compare("ai.onnx.ml") != 0 &&
-          current_opset < opset_list[opset_list.size() - 1] &&
-          current_opset < MINIMUM_SUPPORTED_OPSET)
-        llvm::outs()
-            << "Warning: ONNX " << node.op_type()
-            << " in your model is using Opset " << current_opset
-            << ", which is quite old. Please consider regenerating your "
-               "model with a newer Opset.\n";
-      if (opset_list.size() == 1)
-        return std::string("");
-      for (int i = opset_list.size() - 1; i > 0; i--) {
-        if (current_opset < opset_list[i - 1]) {
-          LLVM_DEBUG(llvm::dbgs() << DEBUG_TYPE << ":   - use Opset "
-                                  << opset_list[i] << "\n");
-          return "V" + std::to_string(opset_list[i]);
-        }
-      }
-    } else {
-      llvm::errs() << node.op_type();
-      if (op_dialect_top_version_map_.find(node.op_type()) !=
-          op_dialect_top_version_map_.end()) {
-        llvm_unreachable(
-            " this Op is not found in the onnx version being used");
-      } else {
-        llvm_unreachable(
-            " this Op is not supported by onnx-mlir's onnx dialect");
```

Comment:
these checks crashed before ImportCustomNode could process custom ops, like FusedMatMul, and non-standard ops with schemas, like TwiceFn in CustomFnTest